src/MercadoPago/Client/Order/OrderClient.php

```diff
@@ -36,4 +38,20 @@ public function create(array $request, ?RequestOptions $request_options = null):
         $result->setResponse($response);
         return $result;
     }
+
+    /**
+     * Method responsible for capturing an order by ID with request options
+     * @param string $orderID ID of Order.
+     * @param \MercadoPago\Client\Common\RequestOptions request options to be sent.
+     * @return \MercadoPago\Resources\Order Order created.
+     * @throws \MercadoPago\Exceptions\MPApiException if the request fails.
+     * @throws \Exception if the request fails.
+     */
+    public function capture(string $orderID, ?RequestOptions $request_options = null): Order
```

Comment:
As variáveis em PHP seguem o padrão `snake_case`
```suggestion
    public function capture(string $order_id, ?RequestOptions $request_options = null): Order
```